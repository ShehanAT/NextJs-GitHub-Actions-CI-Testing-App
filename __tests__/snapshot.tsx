import { render } from '@testing-library/react'
import Home from '@/pages/index'
import Login from '@/pages/login';

it('renders homepage unchanged', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
});

it('renders login page unchanged', () => {
  const { container } = render(<Login />)
  expect(container).toMatchSnapshot()
});
