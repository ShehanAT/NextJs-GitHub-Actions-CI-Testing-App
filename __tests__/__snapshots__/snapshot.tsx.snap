// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main>
      <h1
        class="title"
      >
        Welcome to 
        <a
          href="https://nextjs.org"
        >
          Next.js!
        </a>
      </h1>
      <p
        class="description"
      >
        Get started by editing 
        <code>
          pages/index.js
        </code>
      </p>
      <div
        class="grid"
      >
        <a
          class="card"
          href="https://nextjs.org/docs"
        >
          <h3>
            Documentation →
          </h3>
          <p>
            Find in-depth information about Next.js features and API.
          </p>
        </a>
        <a
          class="card"
          href="https://nextjs.org/learn"
        >
          <h3>
            Learn →
          </h3>
          <p>
            Learn about Next.js in an interactive course with quizzes!
          </p>
        </a>
        <a
          class="card"
          href="https://github.com/vercel/next.js/tree/canary/examples"
        >
          <h3>
            Examples →
          </h3>
          <p>
            Discover and deploy boilerplate example Next.js projects.
          </p>
        </a>
        <a
          class="card"
          href="https://vercel.com/new"
        >
          <h3>
            Deploy →
          </h3>
          <p>
            Instantly deploy your Next.js site to a public URL with Vercel.
          </p>
        </a>
      </div>
    </main>
    <footer
      class="footer"
    >
      <a
        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        Powered by
         
        <span
          class="logo"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2772%27%20height=%2716%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt="Vercel Logo"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
            <noscript />
          </span>
        </span>
      </a>
    </footer>
  </div>
</div>
`;

exports[`renders login page unchanged 1`] = `
<div>
  <form
    autocomplete="off"
    class="makeStyles-container-1"
    novalidate=""
  >
    <div
      class="MuiPaper-root MuiCard-root makeStyles-card-4 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        class="MuiCardHeader-root makeStyles-header-3"
      >
        <div
          class="MuiCardHeader-content"
        >
          <span
            class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
          >
            Login App
          </span>
        </div>
      </div>
      <div
        class="MuiCardContent-root"
      >
        <div>
          <div
            class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink="false"
              for="username"
              id="username-label"
            >
              Username
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input"
                id="username"
                placeholder="Username"
                type="email"
                value=""
              />
            </div>
          </div>
          <div
            class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink="false"
              for="password"
              id="password-label"
            >
              Password
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input"
                id="password"
                placeholder="Password"
                type="password"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiCardActions-root MuiCardActions-spacing"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-loginBtn-2 MuiButton-containedSecondary MuiButton-containedSizeLarge MuiButton-sizeLarge Mui-disabled Mui-disabled"
          disabled=""
          tabindex="-1"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Login
          </span>
        </button>
      </div>
    </div>
  </form>
</div>
`;
